fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -2738,6 +2742,10 @@ show_param ::=
     {:
         RESULT = new ShowDbStmt(parser.wild, parser.where);
     :}
+    | KW_SCHEMAS KW_FROM STRING_LITERAL:catalogName
```

Comment:
use `ident` instead of `STRING_LITERAL`. So we don't need to use `""`.

Reply:
yes, I'm modifying.